cremalab/crema-app-web
# Setup the WDIO configuration

## Reference

[N/A](N/A)

## Overview

Our repo is being setup to use webdriverio for our visual regression tests. It has a powerful regression service (https://github.com/zinserjan/wdio-visual-regression-service) that calculates how alike two individual screenshots are, and allows for us to throw errors when the percentages are too high. In addition, this sets us up for potential gains in regards to adding some automation for accessibility.

### Acceptance Criteria

- [x] WDIO has been introduced to the app
- [x] Standards and conventions for our team are established to help us get going.

## Reviewers' Checklist (go ahead, check 'em)


## Testing Instructions

1. Check out the repo at the branch.
2. Run npm install
3. Run npm run test:wdio
